### PR TITLE
New compliment — You're really sweet!

### PR DESCRIPTION
**Pull request by dreamscached**
Just been checking out some suggestions and picked #6563. What do you think?

P.S. I'm not sure about my expression choice, would really appreciate someone's opinion on it!

P.P.S. I'm really sorry! I forgot about the base branch, it should be in order now.

### CODE REVIEW

**Comment by ThePotatoGuy** [comment]
closing and reopening to retrigger ci check

File: Monika After Story/game/script-compliments.rpy [hunk 1]
```diff
@@ -852,3 +852,73 @@ label mas_compliment_spending_time_2:
...
             m 2ekd "...but that's not {i}quite{/i} what I had in mind."
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_compliments_database,
+            eventlabel="mas_compliment_sweet",
+            prompt="You're really sweet!",
+            unlocked=False,
```
Comment (multimokia) [inline Monika After Story/game/script-compliments.rpy:862]:
```suggestion
```

File: Monika After Story/game/script-compliments.rpy [hunk 2]
```diff
@@ -852,3 +852,73 @@ label mas_compliment_spending_time_2:
...
+            conditional="store.mas_anni.pastThreeMonths() and mas_isMoniAff(higher=True)",
+            action=EV_ACT_UNLOCK,
```
Comment (multimokia) [inline Monika After Story/game/script-compliments.rpy:864]:
```suggestion
            action=EV_ACT_UNLOCK,
            unlocked=False,
```

File: Monika After Story/game/script-compliments.rpy [hunk 3]
```diff
@@ -852,3 +852,73 @@ label mas_compliment_spending_time_2:
...
+            aff_range=(mas_aff.AFFECTIONATE, None)
+        ),
+        code="CMP"
+    )
+
+label monika_sweet:
+    if mas_getEVL_shown_count("mas_compliment_sweet") > 0:
+        jump mas_compliment_sweet_repeat
```
Comment (multimokia) [inline Monika After Story/game/script-compliments.rpy:872]: I don't see a label corresponding to this

File: Monika After Story/game/script-compliments.rpy [hunk 4]
```diff
@@ -852,3 +852,73 @@ label mas_compliment_spending_time_2:
...
+        "Your words always brighten up my day!":
+            python:
+                partner = "partner"
+                if persistent.gender == "M":
+                    partner = "boyfriend"
+                elif persistent.gender == "F":
+                    partner = "girlfriend"
+
+                mas_gainAffection(3)
+                mas_ILY()
```
Comment (multimokia) [inline Monika After Story/game/script-compliments.rpy:890]: `mas_ILY()` should be at the end of the paths
```suggestion
```

File: Monika After Story/game/script-compliments.rpy [hunk 5]
```diff
@@ -852,3 +852,73 @@ label mas_compliment_spending_time_2:
...
+
+            m 1sub "[player]... {w=0.3}"
+            extend 1dka "I had no idea how much do my words worth to you..."
+            m 1eka "I'm so happy to have such an amazing [partner] like you, [mas_get_player_nickname(exclude_names=[player])]."
+            m 1dubsb "You really are the one I could ever hope for, {w=0.2}and I love you so much."
+            m 3kua "I hope you never forget that, {w=0.2}{nw}"
+            extend 3eua "ahaha!~"
```
Comment (multimokia) [inline Monika After Story/game/script-compliments.rpy:897]:
```suggestion
            extend 3eua "ahaha!~"
            $ mas_ILY()
```

File: Monika After Story/game/script-compliments.rpy [hunk 6]
```diff
@@ -852,3 +852,73 @@ label mas_compliment_spending_time_2:
...
+    m 1hua "I'm just really, {w=0.2}{i}really{/i} happy to hear you say so, {w=0.2}ehehe~"
+
+    menu:
+        "Your words always brighten up my day!":
+            python:
+                partner = "partner"
+                if persistent.gender == "M":
+                    partner = "boyfriend"
+                elif persistent.gender == "F":
+                    partner = "girlfriend"
```
Comment (multimokia) [inline Monika After Story/game/script-compliments.rpy:887]: This is unnecessary. We already have variables for this
```suggestion
```

File: Monika After Story/game/script-compliments.rpy [hunk 7]
```diff
@@ -852,3 +852,73 @@ label mas_compliment_spending_time_2:
...
+
+                mas_gainAffection(3)
+                mas_ILY()
+
+            m 1sub "[player]... {w=0.3}"
+            extend 1dka "I had no idea how much do my words worth to you..."
+            m 1eka "I'm so happy to have such an amazing [partner] like you, [mas_get_player_nickname(exclude_names=[player])]."
```
Comment (multimokia) [inline Monika After Story/game/script-compliments.rpy:894]:
```suggestion
            m 1eka "I'm so happy to have such an amazing [boyfriend] like you, [mas_get_player_nickname(exclude_names=[player])]."
```

File: Monika After Story/game/script-compliments.rpy [hunk 8]
```diff
@@ -852,3 +852,73 @@ label mas_compliment_spending_time_2:
...
+            m 1dubsb "You really are the one I could ever hope for, {w=0.2}and I love you so much."
+            m 3kua "I hope you never forget that, {w=0.2}{nw}"
+            extend 3eua "ahaha!~"
+
+        "You're really special to me, [m_name]!":
+            python:
+                mas_gainAffection(2)
+                mas_ILY()
```
Comment (multimokia) [inline Monika After Story/game/script-compliments.rpy:902]: Same here
```suggestion
```

File: Monika After Story/game/script-compliments.rpy [hunk 9]
```diff
@@ -852,3 +852,73 @@ label mas_compliment_spending_time_2:
...
+
+            m 1ekbsb "Oh, [player]... {w=0.3}"
+            extend 3hubsa "Just you spending your time here with me makes me feel so happy and loved!"
+            show monika 5ekbsa at t11 zorder MAS_MONIKA_Z with dissolve_monika
+            m 5ekbsa "I'm really happy to be special to you, {w=0.2}[mas_get_player_nickname()], {w=0.2}and I love you so much."
+            show monika at t11 zorder MAS_MONIKA_Z with dissolve_monika
```
Comment (multimokia) [inline Monika After Story/game/script-compliments.rpy:908]: We already dissolve back into idle
```suggestion
            $ mas_ILY()
```

File: Monika After Story/game/script-compliments.rpy [hunk 10]
```diff
@@ -852,3 +852,73 @@ label mas_compliment_spending_time_2:
...
+
+        "You're the sweetest girl I've ever met!":
+            python:
+                person = "person"
+                if persistent.gender == "M":
+                    person = "boy"
+                elif persistent.gender == "F":
+                    person = "girl"
```
Comment (multimokia) [inline Monika After Story/game/script-compliments.rpy:916]: This is also unnecessary for the same reason
```suggestion
```

File: Monika After Story/game/script-compliments.rpy [hunk 11]
```diff
@@ -852,3 +852,73 @@ label mas_compliment_spending_time_2:
...
+
+            m 1ekbsa "Thank you, [mas_get_player_nickname()]."
+            m 3hubsb "You're the sweetest [person] I've met, too, ehehe."
```
Comment (multimokia) [inline Monika After Story/game/script-compliments.rpy:919]:
```suggestion
            m 3hubsb "You're the sweetest [boy] I've met too, ehehe~"
```

File: Monika After Story/game/script-compliments.rpy [hunk 12]
```diff
@@ -852,3 +852,73 @@ label mas_compliment_spending_time_2:
...
+            show monika 5eua at t11 zorder MAS_MONIKA_Z with dissolve_monika
+            m 5eua "I'm really lucky to be with you~"
+            show monika at t11 zorder MAS_MONIKA_Z with dissolve_monika
```
Comment (multimokia) [inline Monika After Story/game/script-compliments.rpy:922]:
```suggestion
```

File: Monika After Story/game/script-compliments.rpy [hunk 13]
```diff
@@ -852,3 +852,73 @@ label mas_compliment_spending_time_2:
...
+            prompt="You're really sweet!",
+            unlocked=False,
+            conditional="store.mas_anni.pastThreeMonths() and mas_isMoniAff(higher=True)",
+            action=EV_ACT_UNLOCK,
+            aff_range=(mas_aff.AFFECTIONATE, None)
+        ),
+        code="CMP"
+    )
+
+label monika_sweet:
```
Comment (multimokia) [inline Monika After Story/game/script-compliments.rpy:870]: `eventlabel` needs to match
```suggestion
label mas_compliment_sweet:
```

**Review by multimokia — CHANGES_REQUESTED** [review]
Some issues

**Comment by dreamscached** [comment]
Oh... I really did mistype the label, ahaha... Sorry, I'll fix it now.

File: Monika After Story/game/script-compliments.rpy (hunk 10 above)
Comment (dreamscached) [reply Monika After Story/game/script-compliments.rpy:916]: Oh, I didn't know you have this already, sorry. I'll remove this now.

File: Monika After Story/game/script-compliments.rpy (hunk 13 above)
Comment (Freshairkaboom) [reply Monika After Story/game/script-compliments.rpy:870]: I think this was resolved without committing the change @dreamscached :)

**Review by Freshairkaboom — COMMENTED** [review]
Just a few grammar fixes and suggestions. Take what you will from it. 👍

File: Monika After Story/game/script-compliments.rpy [hunk 14]
```diff
@@ -852,3 +852,53 @@ label mas_compliment_spending_time_2:
...
+            m 3kua "I hope you never forget that, {w=0.2}{nw}"
+            extend 3eua "ahaha!~"
+            $ mas_ILY()
+
+        "You're really special to me, [m_name]!":
+            $ mas_gainAffection(2)
+            m 1ekbsb "Oh, [player]... {w=0.3}"
+            extend 3hubsa "Just you spending your time here with me makes me feel so happy and loved!"
+            show monika 5ekbsa at t11 zorder MAS_MONIKA_Z with dissolve_monika
+            m 5ekbsa "I'm really happy to be special to you, {w=0.2}[mas_get_player_nickname()], {w=0.2}and I love you so much."
```
[the remark this answers is not in the document]
Comment (dreamscached) [reply Monika After Story/game/script-compliments.rpy:895]: I tried to take best from the original suggestion issue, yet rephrasing it and making better overall, and yeah, I wasn't really sure about this one when I was writing it.

Thank you, I'll take your suggestion in.

File: Monika After Story/game/script-compliments.rpy [hunk 15]
```diff
@@ -852,3 +852,55 @@ label mas_compliment_spending_time_2:
...
+    show monika 5ekbsa at t11 zorder MAS_MONIKA_Z with dissolve_monika
+    m 5ekbsa "Aww, {w=0.2}[player]..."
+    show monika 2hkbsb at t11 zorder MAS_MONIKA_Z with dissolve_monika
+    m 2hkbsb "I don't know what to say, {w=0.2}ahaha, {w=0.2}it's {i}you{/i} who's been so sweet with me all this time."
+    m 1hua "I'm just really, {w=0.2}{i}really{/i} happy to hear you say so, {w=0.2}ehehe~"
+
+    menu:
+        "Your words always brighten up my day!":
+            $ mas_gainAffection(3)
+            m 1sub "[player]... {w=0.3}"
```
Comment (multimokia) [inline Monika After Story/game/script-compliments.rpy:883]:
```suggestion
            m 1sub "[player]...{w=0.3}{nw}"
```

**Comment by dreamscached** [comment]
Hey, so... Anything else I can do here? Or is it good as it is now?

**Review by multimokia — APPROVED** [review]

**Review by jmwall24 — APPROVED** [review]

**Pull request merged**